app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -7,9 +7,12 @@ class PhoneDeliveryPresenter < TwoFactorAuthCode::GenericDeliveryPresenter
     attr_reader :otp_delivery_preference,
                 :otp_make_default_number,
                 :unconfirmed_phone,
-                :otp_expiration
+                :otp_expiration,
+                :in_multi_mfa_selection_flow,
+                :data
```

Comment:
I'm not expecting `data` to be here, should it be?

```suggestion
                :in_multi_mfa_selection_flow
```

Reply:
You are correct. That's removed.